cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -270,29 +324,37 @@ def parse_harvard_opinions(options: OptionsType) -> None:
         case_name_full = harmonize(data["name"])
 
         citation = cites[0]
-        if skip_processing(citation, case_name, file_path):
```

Comment:
This function is unused now, right? Remove it?

Reply:
This has been removed.